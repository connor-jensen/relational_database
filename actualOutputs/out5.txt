Schedule(D,'113')? Yes(1)
  D='M'