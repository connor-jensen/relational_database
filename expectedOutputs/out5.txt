Schedule(D)? Yes(3)
  D='M'
  D='T'